### DISCUSSION TITLE
Why is typeField no longer automatically added in filters?

### DISCUSSION THREAD

**Discussion by rjmackay** [sensedeep/dynamodb-onetable#221] [Q&A] [answered]
It looks like this was changed in v1.9 but not called out in the release notes. Can you explain why this changed?
It wasn't necessarily an efficient way to filter to just he current types, but it was a great backstop if the query was otherwise ok

https://github.com/sensedeep/dynamodb-onetable/commit/255af1372b4554d6af0a9a8d6d388e3b0bf8b840

**Comment by mobsense** [comment, marked as answer]
Copying the answer from the diff.

I think it was to stop filtering on typeField when using the generic handler. There are such use cases where find must not filter on type.
